### PR TITLE
Update for Hammer-6

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@miq-bot add_label hammer/yes

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit 3961f63c06c2fe7620c81306a723387e52c5bf0c
Author: Adam Grare <agrare@redhat.com>
Date:   Wed May 15 13:55:29 2019 -0400

    Merge pull request #366 from JPrause/hammer6_cl
    
    Update for Hammer-6
    
    (cherry picked from commit 679eebaaabf9b43781dd471303be30ff9baa7edc)
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,11 +5,16 @@ All notable changes to this project will be documented in this file.
...
-## Hammer-5
+## Hammer-6
 
 ### Added
 - Add supports_conversion_host to Host [(#315)](https://github.com/ManageIQ/manageiq-providers-ovirt/pull/315)
 
+### Fixed
+- Fix refreshing ipmi from host [(#359)](https://github.com/ManageIQ/manageiq-providers-ovirt/pull/359)
+
+## Hammer-5
```
Comment (simaishi) [inline CHANGELOG.md]: @JPrause fyi, date is missing.

**Pull request merged**